openapi/paths/reports@kyc-acceptance-summary.yaml

```
@@ -56,13 +56,33 @@ get:
                       type: integer
                       description: Amount of documents that have been accepted automatically.
                       example: 10
+                    manuallyAccepted:
+                      type: integer
+                      description: Amount of documents that have been accepted by user.
+                      example: 30
+                    rejectedAutoAccepted:
+                      type: integer
+                      description: Amount of documents that have been accepted automatically and then rejected by user.
+                      example: 5
+                    acceptedAutoRejected:
+                      type: integer
+                      description: Amount of documents that have been rejected automatically and then accepted by user.
+                      example: 5
+                    totalAccepted:
+                      type: integer
+                      description: Total accepted documents amount.
+                      example: 100
                     total:
                       type: integer
                       description: Total documents amount.
-                      example: 100
-                    autoAcceptedPercentage:
+                      example: 200
+                    accuracyRate:
+                      type: number
+                      description: Percent of automatically accepted and rejected documents that remain in the same status after review.
```

From the numbers above, accuracy rate would be:

= (total - acceptedAutoRejected - rejectedAutoAccepted)/total

With some sample numbers:

1000 total
200 accepted that were auto rejected
20 rejected that were auto accepted
600 total accepted

accuracy rate = (1000 - 200 - 20)/1000 = 780/1000 = 78%
acceptance rate = 600/1000 = 60%

---

(This information may be good in the API description.)
```suggestion
                      description: |
                        Percent of automatically accepted and rejected documents that remain in the same status after review.
                        Calculated by the formula: `(total - accepted auto rejected - rejected auto accepted)/total`.
```